driver/src/main/java/org/kaazing/k3po/driver/internal/netty/bootstrap/http/HttpChildChannelSource.java

```diff
@@ -147,7 +148,10 @@ protected void httpMessageReceived(ChannelHandlerContext ctx, MessageEvent e, Ht
             return;
         }
 
-        Entry<URI, HttpServerChannel> httpBinding = httpBindings.floorEntry(httpLocation);
+        ChannelAddress transportCandidate = Channels.localAddress(ctx.getChannel());
```

Comment:
How do we construct transportCandidate address ? It gives tcp://127.0.0.1:8080 and the bindings has the literal address tcp://localhost:8080